src/bzmesh.jl

```diff
@@ -21,6 +21,46 @@ function bzmesh_uniform(kgrid_size)
 end
 
 
+SymOp = Tuple{Mat3{Int}, Vec3{Float64}}
```

Comment:
Const

Reply:
That works on types, too ... I did not know ...